winui/Rating/Placeholder-value.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Placeholder value in WinUI Rating control | Syncfusion
```

Comment:
Title: Placeholder value in WinUI Rating control | Syncfusion

Reply:
We have follow the small letter for all other controls.